safe-global/safe-wallet-web
# Adjust safe apps tx flow ui

## What it solves

Part of #2067 

## How this PR fixes it

- Updates the Approval Editor design

## How to test it

1. Open a Safe App
2. Create a Swap to see the approval editor
3. Compare to Figma
4. Queue the transaction
5. Go to queue and confirm
6. Compare Approval Editor readonly design

## Screenshots

<img width="715" alt="Screenshot 2023-06-29 at 11 39 37" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/96410e16-c113-4695-8843-39d3730d3d19">
<img width="719" alt="Screenshot 2023-06-29 at 11 59 53" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/f5332ff1-ffde-4daa-8567-dc047da442d9">


## Checklist
